android/runtime/common/src/js/console.js

```diff
@@ -56,13 +72,12 @@ exports.time = function (label = 'default') {
 };
 
 exports.timeEnd = function (label = 'default') {
-	label = `${label}`;
-	const startTime = times.get(label);
-	if (!startTime) {
-		exports.warn(`Label "${label}" does not exist`);
-		return;
+	const warned = logTime(label);
```

Comment:
the use of `warned` here is a little confusing. What you're really determining is if the label existed.